specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-05-01-preview/BillingStatistics.json

```diff
@@ -0,0 +1,233 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/billingStatistics": {
+      "get": {
+        "x-ms-examples": {
+          "Get all billing statistics for a Sentinel workspace.": {
```

Comment:
>Sentinel [](http://example.com/codeflow?start=44&length=8)

What is a sentinel workspace ? THe URL indicates its an OperationalInsights workspace

Reply:
Hi, 
Sentinel is built on Log Analytics workspaces.
All of the sentinel APIs follow the path format of: `providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/{resourceType}` See: [Alert Rules](https://learn.microsoft.com/en-us/rest/api/securityinsights/stable/alert-rules/list?tabs=HTTP), [Sentinel Onboarding States](https://learn.microsoft.com/en-us/rest/api/securityinsights/stable/sentinel-onboarding-states/list?tabs=HTTP)

We will update the description.